From 03f018ec9b2b759d2a99b079e55915616fa1bbe9 Mon Sep 17 00:00:00 2001
From: yhyang2 <yhyang2@nuvoton.com>
Date: Fri, 21 Oct 2022 08:35:47 +0800
Subject: [PATCH 1/1] meta-quanta: meta-olympus-nuvoton: fix firmware update issues
Fix image-u-boot too big and flash permission denied issues

Signed-off-by: yhyang2 <yhyang2@nuvoton.com>
---
 arch/arm/boot/dts/nuvoton-npcm750-runbmc-olympus.dts | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/arch/arm/boot/dts/nuvoton-npcm750-runbmc-olympus.dts b/arch/arm/boot/dts/nuvoton-npcm750-runbmc-olympus.dts
index 1fca68e0a21e..9b6691e54154 100644
--- a/arch/arm/boot/dts/nuvoton-npcm750-runbmc-olympus.dts
+++ b/arch/arm/boot/dts/nuvoton-npcm750-runbmc-olympus.dts
@@ -155,8 +155,7 @@
 			};
 			u-boot@0 {
 				label = "u-boot";
-				reg = <0x0000000 0x80000>;
-				read-only;
+				reg = <0x0000000 0xc0000>;
 			};
 			u-boot-env@100000{
 				label = "u-boot-env";
-- 
2.17.1

